### DISCUSSION TITLE
Install cobbler 3.3.4 in Ubuntu 20.04 TLS failed

### DISCUSSION THREAD

**Discussion by MKCYER** [cobbler/cobbler#3742] [Q&A]
Hi
I tried some ways to install cobbler 3.3.4 in Ubuntu,but all they failed:
1. Download source and make install in Ubuntu 20.04, showed that need additional parameter in setup/py, how can i add it
2. Download deb from build service, use "dpkg -i cobbler_.deb" to install it, and it show "preparing.. unpacking... setting... processing.. ",but nothing happened, i still can't use cobble and cobbler.service not found in /etc/init.d
how can i install cobbler 3.3.4
help

**Comment by SchoolGuy** [comment]
Here are the instructions copied from the [OBS download page](https://software.opensuse.org/download/package?package=cobbler&project=systemsmanagement%3Acobbler%3Arelease33):

```
echo 'deb http://download.opensuse.org/repositories/systemsmanagement:/cobbler:/release33/xUbuntu_20.04/ /' | sudo tee /etc/apt/sources.list.d/systemsmanagement:cobbler:release33.list
curl -fsSL https://download.opensuse.org/repositories/systemsmanagement:cobbler:release33/xUbuntu_20.04/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/systemsmanagement_cobbler_release33.gpg > /dev/null
sudo apt update
sudo apt install cobbler
```

The advantage of using the repository is that you will get automatic bugfix releases. If you don't want to use the repository but install the DEB once then please provide more detailed error descriptions and messages.

**Reply by MKCYER** [reply]
hi, i trid your advice, but i showed the public key is not available: NO_PUBKEY 871920D1991BC93C , so i could not download it

**Reply by SchoolGuy** [reply]
So from the URL that I provided you the key is `ACAA9CF7E6E5A213`. As such I am not sure if this is related to what you executed.

Edit see also here: https://build.opensuse.org/projects/systemsmanagement:cobbler:release33/signing_keys

**Reply by MKCYER** [reply]
I tried it 2 days ago when cobbler was still 3.3.4, the best case was that i stop in redirecting from ip:/cobbler/svc/service.py from client to load kernel, i think it need cobbler-web that i can't install it antway. Then i tried cobbler 3.3.5, but i find some questions that although i set manage_dhcp=true and manage_httpd=true, cobbler sync don't manage them

**Reply by SchoolGuy** [reply]
This sounds like a lot of different issues. One thing that comes to my mind here is that we never had a setting `manage_httpd`, as such I am really confused what the issue in detail now is.
